README.md

```diff
@@ -34,13 +43,17 @@ export default function ButtonVariationProvider({ action }) {
 }
 ```
 
-We also have "consumers" that use the variation provider to perform tasks. Example consumers are:
+Each variation provider will be passed an extras object. It's properties come from the project's configuration, and can be used to get mock data needed to render the variations.
```

Comment:
Being someone still unfamiliar with the extras object, this statement is a little nebulous to me. Where/what is the project's configuration? What type of mock data are we talking about?

I'm thinking we either don't get into the extras object just yet (it is a more advanced feature after all—variations work fine without any extras), or we go into more detail about _why_ extras exist, what problem they solve, and how to set them up. (I'm personally more in favor of not mentioning them up front here in the intro section of the README.)

Reply:
Yeah, it's definitely more advanced. And I haven't covered how to actually integrate this into a project yet. So you're probably right